packages/features/bookings/lib/handleSeats/cancel/cancelAttendeeSeat.ts

```diff
@@ -140,23 +138,11 @@ async function cancelAttendeeSeat(
   );
   await Promise.all(promises);
 
-  const workflowRemindersForAttendee = bookingToDelete?.workflowReminders.filter(
-    (reminder) => reminder.seatReferenceId === seatReferenceUid
-  );
+  const workflowRemindersForAttendee =
+    bookingToDelete?.workflowReminders.filter((reminder) => reminder.seatReferenceId === seatReferenceUid) ??
+    null;
 
-  if (workflowRemindersForAttendee && workflowRemindersForAttendee.length !== 0) {
-    const deletionPromises = workflowRemindersForAttendee.map((reminder) => {
-      if (reminder.method === WorkflowMethods.EMAIL) {
-        return deleteScheduledEmailReminder(reminder.id, reminder.referenceId);
-      } else if (reminder.method === WorkflowMethods.SMS) {
-        return deleteScheduledSMSReminder(reminder.id, reminder.referenceId);
-      } else if (reminder.method === WorkflowMethods.WHATSAPP) {
-        return deleteScheduledWhatsappReminder(reminder.id, reminder.referenceId);
-      }
-    });
-
-    await Promise.allSettled(deletionPromises);
-  }
+  await deleteAllWorkflowReminders(workflowRemindersForAttendee);
```

Comment:
Earlier we were using `Promise.allSettled` which would allow all reminders to be deleted even if one of them errors during deletion.


But `deleteAllWorkflowReminders` is using `Promise.all` now.

Reply:
changed it to `promise.allSettled` in `deleteAllWorkflowReminders` and improved the error message